src/main/resources/db/migration/V1_3__create_table_outgoing_dialogmelding.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE DIALOGMELDING_OUT
+(
+    id                              SERIAL PRIMARY KEY,
+    uuid                            CHAR(36) NOT NULL UNIQUE,
+    created_at                      TIMESTAMPTZ NOT NULL,
+    personident                     VARCHAR(11) NOT NULL,
+    behandler_ref                   CHAR(36), -- TODO: Kan være null?
+    tekst                           TEXT
```

Comment:
Ikke `updatedAt`?

Reply:
Det vet jeg ikke 🤷

Reply:
Jeg og Anders snakket om at vi kan droppe `updated_at` per nå, og så heller introdusere det dersom vi ser at det er en usecase for det (feks identhendelser kunne hatt nytte av dette).